plugins/modules/swim_workflow_manager.py

```diff
@@ -1690,13 +1920,20 @@ def verify_diff_imported(self, import_type):
         """
 
         if import_type == "remote":
-            image_name = self.want.get("url_import_details").get("payload")[0].get("source_url")
-        else:
+            image_names = self.want.get("url_import_details").get("payload")[0].get("source_url")
+        elif import_type == "local":
             image_name = self.want.get("local_import_details").get("file_path")
+        else:
+            image_name = self.want.get("cco_import_details").get("image_name")
```

Comment:
image_names = [self.want.get("cco_import_details", {}).get("image_name")]

Reply:
updated the code